cmd/kubelet/app/server.go

```diff
@@ -239,7 +237,8 @@ HTTP server: The kubelet can also listen for HTTP and respond to a simple API
 					}
 				}
 			}
-
+			// load kubelet config file which will overwrite kubeletConfig
```

Comment:
Very possible I'm missing something, but I'm not seeing how calling `PrintFlags` calls us to overwrite `kubeletConfig`. If it is the case that is overwriting `kubeletConfig`, I think this is a poorly named method, and we will want to think of either how the method could not overwrite `kubeletConfig` or how the method name could make it clear that it has destructive side effects.